### PR TITLE
🔧 Unclickable profile dropdown (When not connected)

### LINKED ISSUE #7459: Profile dropdown options when disconnect is not clickable

### PR DESCRIPTION
**Pull request by daiagi**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [x] Bugfix

## Needs QA check

- @kodadot/qa-guild please review

## Context

- [x] Closes #7459
- [x] From #7375

#### Did your issue had any of the "$" label on it?

- [x] My DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer/?target=1cAsKZYNRb8dkSCpn4eVkEn6ycNZTGoDo5dGDgB8J1UUWK8)

### <samp>🤖 Generated by Copilot at d2322a1</samp>

Improved the appearance and alignment of the profile dropdown menu in the navbar. Changed some props and classes of `o-drop` and its trigger in `components/navbar/ProfileDropdown.vue`.

### <samp>🤖 Generated by Copilot at d2322a1</samp>

> _Oh, we're the crew of the `o-drop` component_
> _We make the profile menu fit and decent_
> _We tweak the `position` and the `class` props_
> _And we add some custom CSS on top_

### CODE REVIEW

File: components/navbar/ProfileDropdown.vue [hunk 1]
```diff
@@ -102,3 +103,15 @@ const toggleLanguageMenu = () => {
...
+<style lang="scss" scoped>
+.menu-mt-0 {
+  :deep(.o-drop__menu) {
+    margin-top: 0;
+  }
+}
+
+.navbar-height {
+  height: 4.5rem;
+}
```
Comment (roiLeo) [inline components/navbar/ProfileDropdown.vue:116]: fixed height, what if we want to change navbar height?

File: components/navbar/ProfileDropdown.vue [hunk 2]
```diff
@@ -102,3 +103,15 @@ const toggleLanguageMenu = () => {
   languageDropdown.value?.toggle()
 }
 </script>
+
+<style lang="scss" scoped>
+.menu-mt-0 {
```
Comment (roiLeo) [inline components/navbar/ProfileDropdown.vue:108]: have you tried `rootClass` or `triggerClass`?

**Review by roiLeo — COMMENTED** [review]
looks like a hotfix

File: components/navbar/ProfileDropdown.vue (hunk 2 above)
Comment (daiagi) [reply components/navbar/ProfileDropdown.vue:108]: I'll check. thanks

File: components/navbar/ProfileDropdown.vue (hunk 1 above)
Comment (daiagi) [reply components/navbar/ProfileDropdown.vue:116]: yes but...
https://github.com/kodadot/nft-gallery/pull/6430
see `components/Navar` line 359
```vue
<style lang="scss" scoped>
:deep(.navbar-explore) {
  .navbar-item {
    height: 4.5rem;
  }
...


}
</style>
```

**Review by roiLeo — APPROVED** [review]
nvm https://github.com/kodadot/nft-gallery/pull/6430#pullrequestreview-1537007456

File: components/navbar/ProfileDropdown.vue (hunk 2 above)
Comment (daiagi) [reply components/navbar/ProfileDropdown.vue:108]: maybe I am misunderstanding you

I want to affect the menu padding
and here is the menuClass
image

Comment (daiagi) [reply components/navbar/ProfileDropdown.vue]: got you
done
https://github.com/kodadot/nft-gallery/pull/7460/commits/ae7a21a7d4a3985a8fc985d1790865fd47b62e3a

**Review by Jarsen136 — APPROVED** [review]

File: components/navbar/ProfileDropdown.vue (hunk 1 above)
Comment (daiagi) [reply components/navbar/ProfileDropdown.vue]: i actually replaced the fixed height with margin (my-4)

**Pull request merged**